src/content/docs/infrastructure/install-infrastructure-agent/manage-your-agent/troubleshoot-running-infrastructure-agent.mdx

```diff
@@ -74,6 +74,28 @@ To change the default settings of `newrelic-infra-ctl`:
         `newrelic-infra-ctl -docker-api-version 1.24`
       </td>
     </tr>
+
+
+    <tr>
+      <td>
+        `container-runtime` ('docker' -default- or 'containerd')
```

Comment:
```suggestion
        `container-runtime` (`docker`, `-default-`, or `containerd`)
```

Reply:
docker is the default one, that's what I mean, maybe we can reformulate.